### PR TITLE
Add more IODA platforms and change the way they are included

### PR DESCRIPTION
**Pull request by danholdaway**
## Description

Add some more IODA platforms to swell and allow for easier addition of platforms not already added.

### CODE REVIEW

File: src/swell/configuration/observation_ioda_names.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  - amsua_metop-a
+  - amsua_metop-b
+  - amsua_metop-c
+  - amsua_n15
+  - amsua_n18
+  - amsua_n19
+  - atms_n20
+  - atms_npp
+  - avhrr3_metop-a
+  - avhrr3_n18
```
Comment (HamidehGMAO) [inline src/swell/configuration/observation_ioda_names.yaml:15]: We should add avhrr3_metop-b here. We can do it when adding the avhrr3_metop-b yaml file too

Comment (danholdaway) [reply src/swell/configuration/observation_ioda_names.yaml:15]: Sounds good. I think that makes sense, to make the change when the YAML gets added.

Comment (gmao-jjin3) [reply src/swell/configuration/observation_ioda_names.yaml:15]: Can you add "amsr2_gcom-w1" as well? Thanks.

**Review by gmao-jjin3 — APPROVED** [review]

**Comment by HamidehGMAO** [comment]
It works fine Dan. Thanks for the fix!

**Review by HamidehGMAO — APPROVED** [review]

File: src/swell/configuration/observation_ioda_names.yaml (hunk 1 above)
Comment (danholdaway) [reply src/swell/configuration/observation_ioda_names.yaml:15]: Done

File: src/swell/configuration/observation_ioda_names.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  - avhrr3_n19
+  - cris-fsr_n20
+  - cris-fsr_npp
+  - gpm_gmi
```
Comment (gmao-jjin3) [inline src/swell/configuration/observation_ioda_names.yaml:19]: It should be  "- gmi_gpm".

**Review by gmao-jjin3 — COMMENTED** [review]
Need change "gpm_gmi" to "gmi_gpm".

**Pull request merged**